src/main/java/com/mojang/brigadier/arguments/StringArgumentType.java

```diff
@@ -13,7 +13,7 @@
 public class StringArgumentType implements ArgumentType<String> {
     private final StringType type;
 
-    private StringArgumentType(final StringType type) {
+    StringArgumentType(final StringType type) {
```

Comment:
Would not matter much, but what do you think about using `protected` as visibility instead? This would allow subclasses outside of this package (which should possibly be the case for the other argument types as well, but that is a different topic).

Reply:
agreed, fixed